src/content/app-platform/app-bundle/index.md

```diff
@@ -0,0 +1,240 @@
+---
+linkTitle: App Bundle
+title: App Bundle reference
+description: Overview of the the App Bundle concept, how it works, and how to configure it.
+menu:
+  main:
+    parent: app-platform
+weight: 30
+aliases:
+ - /reference/app-bundle/
+last_review_date: 2022-09-14
```

Comment:
Aliases should only be created if a page is moved, to have a redirect from the old URL (the alias) to the new URL.

```suggestion
weight: 30
last_review_date: 2022-09-14
```